### PR TITLE
one_vm: fix for 'NoneType' object has no attribute 'split' in get_vm_labels_and_attributes_dict

### PR DESCRIPTION
**Pull request by d34d5p4rr0w**
##### SUMMARY
Got a bug with NoneTypee when parsing VM's list, create small fix for this
'NoneType' object has no attribute 'split'

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
community.general.one_vm.py

##### ADDITIONAL INFORMATION
VM without labels and caused an exception in function 
get_vm_labels_and_attributes_dict 
AttributeError: 'NoneType' object has no attribute 'split'

```
File \"/tmp/ansible_community.general.one_vm_payload_ko5loyc4/ansible_community.general.one_vm_payload.zip/ansible_collections/community/general/plugins/modules/cloud/opennebula/one_vm.py\", line 993, in get_vm_labels_and_attributes_dict\nAttributeError: 'NoneType' object has no attribute 'split'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
```
After fix everything is fine

### CODE REVIEW

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks!

**Comment by d34d5p4rr0w** [comment]
> Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks!

Added changelog

File: changelogs/fragments/5489-nonetype-in-get-vm-by-label.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - one_vm - NoneType object has no attribute split in get_vm_labels_and_attributes_dict (https://github.com/ansible-collections/community.general/pull/5489).
```
Comment (felixfontein) [inline changelogs/fragments/5489-nonetype-in-get-vm-by-label.yml:2]:
```suggestion
  - one_vm - avoid splitting labels that are ``None`` (https://github.com/ansible-collections/community.general/pull/5489).
```

**Comment by felixfontein** [comment]
Can you fix the line endings in the changelog fragment (Windows style → Unix style)?

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@d34d5p4rr0w thanks for fixing this!

**Pull request merged**